polypie.py

```diff
@@ -55,16 +52,14 @@ def _call_func(func_key, args=None, kwargs=None):
 
 
 def polymorphic(func):
-    global _registry
     func_key = func.__module__ + '.' + func.__qualname__
     parameters = signature(func).parameters
     parameters_tuple = tuple(
         HashableParameter.from_parameter(p) for p in parameters.values())
     if func_key not in _registry:
+        @wraps(func)
```

Comment:
It also changes `__module__` attribute (to `polypie`) and assigns wrapped function to `__wrapped__` attribute.